README.md

```diff
@@ -6,19 +6,19 @@ The following directory structure is created:
 
 ```
 /etc
-    /greenboot.d
-                /check
-                      /required
-                      /wanted
-                /green
-                /red
+    /greenboot
+              /check
+                    /required.d
+                    /wanted.d
+              /green.d
+              /red.d
```

Comment:
have you ever used `/usr/bin/tree` before? putting the lines in here like that utility does might be nice.

Reply:
will check that out!